SDK/MobileCenter/Microsoft.Azure.Mobile.Windows.Shared/Storage/Storage.cs

```diff
@@ -235,7 +207,7 @@ internal Storage(IStorageAdapter storageAdapter)
             }
```

Comment:
In general I try to avoid catch-alls, but perhaps we can here since all this does is wrap and re throw the exception